### ISSUE TITLE
Any idea how long each guest token is valid for?

### ISSUE THREAD

**Issue by abhranil26** [fa0311/TwitterInternalAPIDocument#660]
I am using endpoints which can be viewed in incognito mode. I see each ip has a 95 request limit for a 13 minute window. But any idea how long a guest token remains valid before it starts giving 403?
I am caching the guest token in order to minimize requests but in production ended up getting 403 errors.
any idea @fa0311 ?

**Comment by fa0311** [comment]
<img width="607" alt="image" src="https://github.com/user-attachments/assets/c614f4aa-c6c8-44c6-b25e-66542ce1fee9">
Perhaps you do, you may not last a day.
It's 16:40 JST on October 14.

**Comment by fa0311** [comment]
I do not know how the official client behaves and renews the token when the token expires
I need to investigate.

**Comment by abhranil26** [comment]
yeah, i tried researching a bit i dont think guest token remain valid for more than 30 mins, even [perplexity](https://www.perplexity.ai/search/how-long-are-twitter-guest-tok-j6VKGyQAQAOi.dMEgaI5lQ) says it might be 15 mins max.
